python3/36.py

```diff
@@ -0,0 +1,53 @@
+class Solution:
+    def isValidSudoku(self, board: List[List[str]]) -> bool:
```

Comment:
**PaiR(PR AI Reviewer) :** Using a tuple for indexing in `boxMap` can be confusing. Consider using a separate variable to store the box index.